grpc/proto/v1/tenants.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package weaviate.v1;
+
+option go_package = "github.com/weaviate/weaviate/grpc/generated;protocol";
+option java_package = "io.weaviate.client.grpc.protocol.v1";
+option java_outer_classname = "WeaviateProtoTenants";
+
+enum TenantActivityStatus {
+  TENANT_ACTIVITY_STATUS_UNSPECIFIED = 0;
+  TENANT_ACTIVITY_STATUS_HOT = 1;
+  TENANT_ACTIVITY_STATUS_COLD = 2;
+  TENANT_ACTIVITY_STATUS_WARM = 3;
+  TENANT_ACTIVITY_STATUS_FROZEN = 4;
+}
+
+message TenantsGetRequest {
+  string collection = 1;
+  bool is_consistent = 2;
+  // we might need to add a tenant-cursor api at some point, make this easily extendable
+  oneof params {
+    TenantNames names = 3;
```

Comment:
can this be `oneof string names` ?

Reply:
[No it cannot!](https://github.com/protocolbuffers/protobuf/issues/2592#issuecomment-284540212)